app/operations/orders/create.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Orders
+  class Create < Callable
+    def initialize(user_id = nil, params = {})
```

Comment:
what's the point of `user_id = nil`? Create some mechanism to prevent creation of orders without user.